src/test/java/io/qase/app/CreateUpdateProjectTest.java

```diff
@@ -0,0 +1,100 @@
+package io.qase.app;
+
+import com.github.javafaker.Faker;
+import io.qase.app.dto.Project;
+import io.qase.app.page.LoginPage;
+import io.qase.app.page.ProjectPage;
+import io.qase.app.page.ProjectsPage;
+import io.qase.app.page.SettingsPage;
+import io.qase.app.step.ProjectSteps;
+import org.testng.annotations.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CreateUpdateProjectTest extends BaseTest {
+
+    Project project;
+    List<Project> projects;
```

Comment:
you can create list of project names (List<String) and put created project name in the list at the end of the test (directly in test) and then delete projects by names